protoutil/txutils.go

```diff
@@ -202,7 +202,7 @@ func createTx(
 		}
 
 		if !bytes.Equal(a1, r.Payload) {
-			return nil, errors.New("ProposalResponsePayloads do not match")
```

Comment:
Maybe print a base64 encoding of both a1 and r.Payload?

Reply:
The caller has being catching the error and appending "- proposal response: ", which is `a1`.

I'm new to golang. Would you show me a method to call for base64 encoding?